code/src/main/java/com/onlinecontacttracing/servlets/get-data/NegativeUserInfoServlet.java

```diff
@@ -0,0 +1,48 @@
+package com.onlinecontacttracing.authentication;
+
+import com.google.api.client.auth.oauth2.Credential;
+import com.google.api.client.googleapis.auth.oauth2.GoogleIdToken.Payload;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import static com.googlecode.objectify.ObjectifyService.ofy;
+import com.onlinecontacttracing.storage.NegativeUser;
+import java.util.Optional;
+
+@WebServlet("/get-negative-user-info")
+public class NegativeUserInfoServlet extends CheckForApiAuthorizationServlet {
+
+  String getServletURIName() {
+    return "/get-negative-user-info";
+  }
+  
+  void useCredential(String userId, Credential credential, HttpServletResponse response) throws IOException, InterruptedException {
+    Thread contactInfo = new Thread(new CalendarDataForNegativeUser(credential));
+
+    contactInfo.start();
+  
+    contactInfo.join();
+
+    response.sendRedirect("/?page=notification");
+  }
+
+
+
```

Comment:
here and elsewhere: 1 line between methods please,

Reply:
fixed